scripts/bench/bench.sh

```
@@ -0,0 +1,58 @@
+#   discs.txt                      one "name|path" per line; path is a disc folder or .iso
+#   stat.py  evict.py              copied from this directory
+# Output: results.tsv and out/<disc>/<tool>-<rep>/{report.txt,log}.
+set -u
+B=${BENCH_DIR:-/tmp/bdinfo-bench}
+BIN=$B/bin; OUT=$B/out
+OFF=${BDINFO_OFFICIAL_BIN:-/root/github/oss/bdinfo-official/bdinfo_linux_v2.0.5_extracted/BDInfo}
+REPS=${REPS:-3}
+RES=$B/results.tsv
+printf 'disc\ttool\trep\twall_s\tuser_s\tsys_s\tmaxrss_kb\trc\n' > "$RES"
+
+drop() { sync; python3 $B/evict.py "$1"; }
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)